### DISCUSSION TITLE
oh-my-posh not work on powershell administrator mode

### DISCUSSION THREAD

**Discussion by superlee007** [JanDeDobbeleer/oh-my-posh#5977] [Troubleshoot]
I have a question. I installed Oh My Posh on my Windows PowerShell by following the [official installation guide](https://ohmyposh.dev/docs/installation/windows). I used `winget install JanDeDobbeleer.OhMyPosh -s winget` to install Oh My Posh and then proceeded to install the recommended fonts. The guide suggests using the command `oh-my-posh font install meslo` to install fonts, but due to network issues, I kept getting timeout errors when accessing GitHub.

As a workaround, I manually downloaded the Meslo font from [this GitHub link](https://github.com/ryanoasis/nerd-fonts/releases/latest/download/meslo.zip), extracted the files, and installed the fonts by copying them to C:\Windows\Fonts. Next, I configured PowerShell's font settings using the following JSON:

```json
{
    "profiles": {
        "defaults": {
            "font": {
                "face": "MesloLGM Nerd Font"
            }
        }
    }
}
```
Finally, I ran `notepad $PROFILE` and added `oh-my-posh init pwsh | Invoke-Expression`. After restarting PowerShell, the theme and style appeared correctly.
![image](https://github.com/user-attachments/assets/067b6890-992e-47f0-9ae9-828d7a9ad782)

However, when I open PowerShell as an administrator, the font is not applied correctly and displays garbled text. I have to manually set the font to MesloLGM Nerd Font Mono in the properties every time I start PowerShell in administrator mode. How can I fix this so the font applies automatically for administrator PowerShell?

![image](https://github.com/user-attachments/assets/0c1608c2-a48a-4ef0-8093-d510fa5af2ec)

**Comment by JanDeDobbeleer** [comment]
@superlee007 you're starting administrator in a different terminal so that one needs to be configured as well. Administrator is a different user, different settings apply.

**Reply by fangbuilt** [reply]
uh yes, how?

**Reply by JanDeDobbeleer** [reply]
open the `$PROFILE` there and set up oh-my-posh just like you would do elsewhere.
